zrml/swaps/src/lib.rs

```diff
@@ -1392,9 +1391,10 @@ mod pallet {
                                 &pool_account,
                                 subsidy,
                             )?;
-                            total_balance = subsidy;
+                            total_balance = total_balance.saturating_add(subsidy);
```

Comment:
This changes nothing, because for the first account adding and setting are equivalent.

Reply:
Good catch, thanks!